atlas/templates/Makefile.kind.gotmpl

```diff
@@ -0,0 +1,146 @@
+KIND_NAME        := minikube
+KUBECTL_VERSION  ?= v1.15.10
+KIND_VERSION     ?= v0.8.1
+NODE_VERSION     ?= 1.19.0
+DAPR_VERSION     := 1.1.1
+HELM_VERSION     := v3.5.3
+IMAGE_VERSION    := $(shell git describe --long --tags --dirty=-unreleased --always || echo pre-commit)
+REGISTRY         := {{ .Registry }}
+IMAGE_REPO       := {{ .Name }}
+DOCKER_IMAGE     := $(REGISTRY)/$(IMAGE_REPO):$(IMAGE_VERSION)
+NODE_IMAGE       ?= kindest/node:v${NODE_VERSION}
+KUBECONFIG       ?= ${HOME}/.kube/config
+KUBECTL          := bin/kubectl
+KIND             := bin/kind
+HELM             := bin/helm
+TARGET_NAMESPACE ?= `cat kind/.id`
```

Comment:
Put this file in the root, it's used generally for `helm` usecases and isn't specific to kind

```suggestion
TARGET_NAMESPACE ?= `cat .id`
```